guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -0,0 +1,230 @@
+[id="upgrading-{project-context}-or-proxy-in-place-using-leapp_{context}"]
+= Upgrading {Project} or {SmartProxy} to {EL} 9 in-place by using Leapp
+
+{Project} and {SmartProxy} are supported on both {EL}8 and {EL}9.
+You can upgrade your {Project} and {SmartProxy} operating system from {EL} 8 to {EL} 9. 
+When upgrading your {Project} or {SmartProxy} installation from {EL} 8 to {EL} 9, use the Leapp tool to upgrade as well as to help detect and resolve issues that could prevent you from upgrading successfully.
+
+.Prerequisites
+ifdef::satellite[]
+* Review Known Issues before you begin an upgrade.
+For more information, see {ReleaseNotesDocURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+endif::[]
+* {Project} {ProjectVersion} or {SmartProxy} {ProjectVersion} running on {EL} 8.
+ifndef::satellite[]
+* Access to available repositories or a local mirror of repositories.
+endif::[]
+
+ifeval::["{mode}" == "disconnected"]
+* You require access to {RHEL} and {Project} packages.
+Obtain the ISO files for {RHEL}{nbsp}9 and {Project}.
+For more information, see xref:Upgrading_satellite_upgrading-disconnected[].
+* Since Leapp completes part of the upgrade in a container that has no access to additional ISO mounts, the repositories cannot be served from a locally mounted ISO but must be delivered over the network from a different machine.
+endif::[]
+
+.Procedure
+ifndef::satellite[]
+. Configure the repositories to obtain Leapp.
++
+Configure the https://copr.fedorainfracloud.org/coprs/g/theforeman/leapp/[@theforeman/leapp COPR Repository], which contains Leapp packages with patches to support {Project} or {SmartProxy} upgrades:
++
+----
+# dnf copr enable @theforeman/leapp
+----
+Note that this is not required for {RHEL} based installations.
```

Comment:
What does "RHEL-based" mean? Any {EL} distribution? Or just RHEL?

Reply:
This step is not required on RHEL as RHEL does carry the necessary leapp packages.
On any other distro (CentOS, Rocky, Alma), it is required.

Reply:
@Lennonka should we keep this as `{RHEL} based`, or should we modify this?

Reply:
Just remove the word "based", otherwise we're good.